eved/cmd/root.go

```diff
@@ -200,7 +200,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		genutilcli.ValidateGenesisCmd(app.ModuleBasics),
 		AddGenesisAccountCmd(app.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
-		//		NewTestnetCmd(app.ModuleBasics, banktypes.GenesisBalancesIterator{}),
+		testnetCmd(app.ModuleBasics, banktypes.GenesisBalancesIterator{}),
```

Comment:
We removed this since we are going to have to test with docker / compose

also
eved testnet init
Error: unable to resolve type URL /cosmos.staking.v1beta1.MsgCreateValidator

built in testnet is too hard to add custom logic for, might as well just run natively